cmd/utils_test.go

```diff
@@ -354,3 +354,37 @@ func Test_sliceContains_Fail(t *testing.T) {
 		t.Errorf("Slice a should not have b")
 	}
 }
+
+func Test_generateGraph(t *testing.T) {
+
+	/*
+
+	         A
+	       / | \
+	      /  |  \
+	     G   B   D
+	     |\  |   /\
+	     | \ |  /  \
+	     F   C      E
+
+	 A is the main Module here
+	 */
+	//x@version format signifies x-->module name,version--> version of module
```

Comment:
I think we can do away with this too :)

Reply:
When you say do away you mean remove the comments or the x@version format for the gomod graph output ?

Reply:
I mean we can remove line 370 and 372 🙂

Reply:
The versions in the test are actually a very good addition! Thanks for adding them!!